metadata-models/src/main/pegasus/com/linkedin/ml/metadata/QuantitativeAnalyses.pdl

```diff
@@ -0,0 +1,17 @@
+namespace com.linkedin.ml.metadata
+
+/**
+ * Quantitative analyses should be disaggregated, that is, broken down by the chosen factors. Quantitative analyses should provide the results of evaluating the MLModel according to the chosen metrics, providing confidence interval values when possible.
+ */
+record QuantitativeAnalyses {
```

Comment:
It's nice to link the results to a dashboard. But maybe we should also capture the value of each metric provided in `Metrics` aspect?

Reply:
As the `DashboardType` don't exist yet I have referred to a `ResultsType` union. We could add it to the union once we have the `DashboardType`. 
I don't understand your comment about the `Metrics` aspect.

Reply:
So some results can have a score associated with them. We can capture the scores as well in this aspect, perhaps as `<metric, value>` pair, where value is a typefef union of int, string, boolean, etc. What do you think?

Reply:
It's fine to leave it now, we can add it later if need be.